src/pages/mission/goal/WeeklyGoalAchievement.tsx

```
@@ -0,0 +1,158 @@
+                  <div className="flex items-center gap-10">
+                    <span className="body-2 text-black">
+                      +{weeklyBonusAcorns}
+                    </span>
+                    <IcColoredAcorn className="h-24 w-24" />
+                  </div>
+                </div>
+
+                {questionResults.map((result, index) => (
+                  <div
+                    key={index}
+                    className="flex items-center justify-between py-8 pl-10"
```

Biome (2.3.13)

[error] Avoid using the index of an array as key property in an element.

This is the source of the key value.

The order of the items may change, and this also affects performances and component state.
Check the React documentation. 

(lint/suspicious/noArrayIndexKey)